modules/ROOT/attachments/schema-registry-api.json

```diff
@@ -987,6 +1008,13 @@
             "in": "query",
             "required": false,
             "type": "string"
+          },
+          {
+            "name": "format",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "If set to an empty string, returns the schema in the default (current) format. For Protobuf only: if set to 'serialized', returns the schema in its wire binary format in Base64."
```

Comment:
@IoannisRP for the 200 response for this endpoint, the schema is currently set to "type": "string", would it not be a $ref to schema_def instead?